src/InformaticsGateway/Services/Http/MonaiAeTitleController.cs

```diff
@@ -240,7 +231,7 @@ private async Task ValidateCreateAsync(MonaiApplicationEntity item)
             {
                 throw new ObjectExistsException($"A MONAI Application Entity with the same AE Title '{item.AeTitle}' already exists.");
             }
-            if (item.IgnoredSopClasses.Any() && item.AllowedSopClasses.Any())
+            if (item.IgnoredSopClasses.Count != 0 && item.AllowedSopClasses.Count != 0)
```

Comment:
For consistency do you want to change this to `is not` as well instead of `!=`. Happy either way

Reply:
good point, will  do.